content/events/23-24/linux_party.md

```diff
@@ -11,4 +11,8 @@ tags:
   - Linux
 ---
 
-> **Meer info komt later**
+Zeus helpt elk jaar de nieuwe (en minder nieuwe) informaticastudenten met het installeren van Linux op hun laptops.
+
+Ben je niet zeker hoe het allemaal werkt, of waar te beginnen? Heb je een poging ondernomen maar is ineens alles kapot? Of wil je wat hulp om zaken te verbeteren? In elk geval verwelkomt Zeus je donderdag 7 maart in onze kelder, gebouw S9, wij helpen je verder!
+
+Tot dan!
```

Comment:
Dit willen we waarschijnlijk nog uitbereiden met meer instructies:

Of linken naar dezelfde blogpost als hier, https://628.pr.zeus.gent/events/21-22/02_lip/ of gelijkaardige uitleg. Dat mensen een beetje voorbereid komen